ext/pcre/config0.m4

```diff
@@ -11,21 +11,10 @@ PHP_ARG_WITH(pcre-jit,,[  --with-pcre-jit         Enable PCRE JIT functionality
 
   if test "$PHP_PCRE_REGEX" != "yes" && test "$PHP_PCRE_REGEX" != "no"; then
 
+    # FIXME: don't define "I'm not bundled" as only versions installed in /usr
```

Comment:
Not sure what the best way to do this is. PCRE is always present and should default to the bundled lib. Maybe replace the existing `--with-pcre-regex=DIR` by a `--disable-bundled-pcre` flag?